sdk/metricsadvisor/ai-metrics-advisor/test/public/advisorclient.spec.ts

```diff
@@ -590,7 +590,7 @@ matrix([[true, false]] as const, async (useAad) => {
           }
         });
 
-        it("retrieves Anomaly feedback", async function() {
+        it("retrieves Comment feedback", async function() {
```

Comment:
would this change requires a new recording? please ensure the old recording is deleted.

Reply:
Yes the old recordings are deleted, and the new recordings are there for both browser and node